lib/SIL/Bridging.cpp

```diff
@@ -159,7 +159,9 @@ Type TypeConverter::getLoweredCBridgedType(AbstractionPattern pattern,
 
   // Class metatypes bridge to ObjC metatypes.
   if (auto metaTy = t->getAs<MetatypeType>()) {
-    if (metaTy->getInstanceType()->getClassOrBoundGenericClass()) {
+    if (metaTy->getInstanceType()->getClassOrBoundGenericClass()
```

Comment:
Can any other case come up here?

Reply:
Shouldn't be. Only class-constrained metatypes are considered representable in ObjC at the AST level.